README.md

```diff
@@ -181,16 +160,20 @@ The following code will transfer 20 KIN to the recipient account "GDIRGGTBE3H4CU
 ```java
 String toAddress = "GDIRGGTBE3H4CUIHNIFZGUECGFQ5MBGIZTPWGUHPIEVOOHFHSCAGMEHO";
 BigDecimal amountInKin = new BigDecimal("20");
+int fee = 100 // we could write some custom fee or we can can call the blockchain in order to retreive
```

Comment:
I think it's better to have it above, and generally to add a short explanation for giving larger context, that there're 2 possible ways (with fee, and without using your application - whitelist)